src/Kentico.Xperience.Lucene.Core/Indexing/DefaultLuceneTaskProcessor.cs

```diff
@@ -101,7 +101,7 @@ private async Task ProcessLuceneBatch(IEnumerable<LuceneQueueItem> queueItems, L
             }
             catch (Exception ex)
             {
-                eventLogService.LogError(nameof(DefaultLuceneTaskProcessor), nameof(ProcessLuceneTasks), ex.Message);
+                eventLogService.LogException(nameof(DefaultLuceneTaskProcessor), nameof(ProcessLuceneTasks), ex);
```

Comment:
We weren't logging the exception which could make it very difficult to identify the cause of the issue.